hawc/apps/invitro/exports.py

```diff
@@ -1,362 +1,622 @@
+                ],
+                columns=list(rob_headers.values()),
+                index=study_ids,
+            )
+            df = df.join(rob_df, on="study-id")
+
+        df["key"] = df["iv_endpoint-id"]
+
+        df = self.handle_dose_groups(df)
+        df = self.handle_benchmarks(df)
+        df = self.handle_dsstox(df)
+        df = self.handle_categories(df)
+
+        df = df.rename(
+            columns={
+                "study-id": "study id",
+                "study-hero_id": "study hero_id",
+                "study-pubmed_id": "study pubmed_id",
+                "study-doi": "study doi",
+                "study-short_citation": "study name",
+                "study-study_identifier": "study identifier",
+                "study-published": "study published",
+            }
+        )
+        df = df.rename(
+            columns={
+                "iv_chemical-id": "chemical id",
+                "iv_chemical-name": "chemical name",
+                "iv_chemical-cas": "chemical CAS",
+                "iv_chemical-purity": "chemical purity",
+                "iv_experiment-id": "IVExperiment id",
+                "iv_experiment-dose_units": "Dose units",
+                "iv_experiment-metabolic_activation": "Metabolic activation",
+                "iv_experiment-transfection": "Transfection",
+                "iv_cell_type-id": "IVCellType id",
+                "iv_cell_type-species": "cell species",
+                "iv_cell_type-strain": "cell strain",
+                "iv_cell_type-sex": "cell sex",
+                "iv_cell_type-cell_type": "cell type",
+                "iv_cell_type-tissue": "cell tissue",
+                "iv_endpoint-id": "IVEndpoint id",
+                "iv_endpoint-name": "IVEndpoint name",
+                "iv_endpoint-effects": "IVEndpoint description tags",
+                "iv_endpoint-assay_type": "assay type",
+                "iv_endpoint-short_description": "endpoint description",
+                "iv_endpoint-response_units": "endpoint response units",
+                "iv_endpoint-observation_time": "observation time",
+                "iv_endpoint-observation_time_units": "observation time units",
+                "iv_endpoint-NOEL": "NOEL",
+                "iv_endpoint-LOEL": "LOEL",
+                "iv_endpoint-monotonicity": "monotonicity",
+                "iv_endpoint-overall_pattern": "overall pattern",
+                "iv_endpoint-trend_test": "trend test result",
+            }
+        )
 
-        return rows
+        return df
+
+
+class InvitroGroupExporter(Exporter):
+    def build_modules(self) -> list[ModelExport]:
+        return [
+            StudyExport(
+                "study",
+                "experiment__study",
+                include=("id", "short_citation", "study_identifier", "published"),
+            ),
+            IVChemicalExport(
+                "iv_chemical",
+                "chemical",
+            ),
+            DSSToxExport(
```

Comment:
I think `DSSToxExport` can be removed, and we add `dtxsid_id` to the `IVChemicalExport`.